README.md

```diff
@@ -390,6 +390,24 @@ Artistic License
 * `artistic2`
 
 Artistic License 2.0
+* `ccby`
```

Comment:
How do you come up with these abbreviations? Are they common or standard? Thanks!